app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    @greeting = Greeting.order('RANDOM()').first
+    render json: @greeting
+  end
+end
```

Comment:
- [ ] rendering the random greeting doesn't work, please make sure that a user sees the random greeting once he or she visited the correct routes as well as get the random greeting.
![data](https://user-images.githubusercontent.com/55635977/140905174-f1a47ce6-d6cb-4948-89f4-0947971cc703.png)

Reply:
Hello @Emmyn5600, the API works directly on the root page, you don't need to go to another path. Kindly have another look

Reply:
@ZeenatLawal it doesn’t work when  I visit the root path as well

Reply:
![data1](https://user-images.githubusercontent.com/55635977/140907305-f5a7bcf8-c1ff-491c-9caa-23313ce9e60c.png)